README.md

```diff
@@ -66,17 +66,32 @@ The [Lighthouse](https://github.com/submariner-io/lighthouse) project helps in c
 
 To deploy Submariner with Lighthouse follow the below steps.
 
-* Deploy Broker with Dataplane - To deploy the Submariner broker along with the data cluster, run using the broker-info.subm generated in the folder from which the previous step was run.
+* Create a kubeconfig file which has the context of all the clusters (including the Broker cluster). If KUBECONFIG environment variable set with the path to all clusters (like KUBECONFIG=/home/user/.config/east/auth/kubeconfig-dev:/home/user/.config/west/auth/kubeconfig-dev), you could use the below command.
```

Comment:
Specifying "If KUBECONFIG environment variable set with...", is not optional, but mandatory. Please add:
export KUBECONFIG="${KUBECONFIG_BROKER}:${KUBECONFIG_A}:${KUBECONFIG_B}"

Reply:
No, it’s not mandatory, there are other ways to create a merged file. However, being directive is better in end-user documentation, so we could rephrase this to say “export ...” with no conditional.

Reply:
Done.